assets/javascript/app.js

```diff
@@ -179,158 +177,177 @@ function tvQuery() {
+                        img.data("title", title);
+                        //Add data-value for overview
+                        img.data("synopsis", overview);
+                        // Append image to the new div
+                        img.appendTo(newDiv);
+                        // Append new divs to #recommendation
+                        finalRecDiv.appendTo('#recommendation');
+                        newDiv.appendTo('#recommendation');
+                        clickPoster.appendTo('#recommendation');
+                        // Store Movie Title and Synopsis to global variables to display on-click
+                        movieTitle.push(title);
+                        movieSynopsis.push(overview);
+                    }
 
-            $('.carousel').carousel();
-        }
+                    $('.carousel').carousel();
+                }
 
-        $(document.body).on("click", "#movie-1", function() {
-            $("#recommendation").empty();
-            // Hook into contentDiv
-            // Adds title and overview to DOM 
-            $("#recommendation").html("<h4>" + movieTitle[0] + "</h4><p>" + movieSynopsis[0] + "</p>")
-        });
-        $(document.body).on("click", "#movie-2", function() {
-            $("#recommendation").empty();
-            // Hook into contentDiv
-            // Adds title and overview to DOM 
-            $("#recommendation").html("<h4>" + movieTitle[1] + "</h4><p>" + movieSynopsis[1] + "</p>")
-        });
-        $(document.body).on("click", "#movie-3", function() {
-            $("#recommendation").empty();
-            // Hook into contentDiv
-            // Adds title and overview to DOM 
-            $("#recommendation").html("<h4>" + movieTitle[2] + "</h4><p>" + movieSynopsis[2] + "</p>")
-        });
-        resetGlobals();
+                $(document.body).on("click", "#movie-1", function () {
+                    $("#recommendation").empty();
+                    // Hook into contentDiv
+                    // Adds title and overview to DOM 
+                    $("#recommendation").html("<h4>" + movieTitle[0] + "</h4><p>" + movieSynopsis[0] + "</p>")
+                });
+                $(document.body).on("click", "#movie-2", function () {
+                    $("#recommendation").empty();
+                    // Hook into contentDiv
+                    // Adds title and overview to DOM 
+                    $("#recommendation").html("<h4>" + movieTitle[1] + "</h4><p>" + movieSynopsis[1] + "</p>")
+                });
+                $(document.body).on("click", "#movie-3", function () {
+                    $("#recommendation").empty();
+                    // Hook into contentDiv
+                    // Adds title and overview to DOM 
+                    $("#recommendation").html("<h4>" + movieTitle[2] + "</h4><p>" + movieSynopsis[2] + "</p>")
+                });
+                resetGlobals();
 
+                $("#recommendation").on('click', function () {
```

Comment:
Rather than wrapping these in on-click events, can we put this in its own function (i.e. `calliTunes();`)? I would like to call this at the same time as writing the title and synopsis to the DOM, so there won't be a need for the on-click.